fdbserver/OldTLogServer_6_2.actor.cpp

```diff
@@ -2813,7 +2813,10 @@ ACTOR Future<Void> restorePersistentState(TLogData* self,
 		removed.push_back(errorOr(logData->removed));
 		logsByVersion.emplace_back(ver, id1);
 
-		TraceEvent("TLogPersistentStateRestore", self->dbgid).detail("LogId", logData->logId).detail("Ver", ver);
+		TraceEvent("TLogPersistentStateRestore", self->dbgid)
+			.detail("LogId", logData->logId)
+			.detail("Ver", ver)
+			.detail("RecoveryCount", std::to_string(logData->recoveryCount));
```

Comment:
to_string is not necessary here. Same for the 2 changes below.

Reply:
It seems that we have to use to_string since DBRecoveryCount is a uint64_t.

Reply:
1. It seems unnecessary to add this info to the older TLogServers @xumengpanda.